Falanx.Proto.Generator/CreateProto.fs

```diff
@@ -71,7 +71,13 @@ module Proto =
               yield "Microsoft.FSharp.Control"
               yield "Microsoft.FSharp.Text" ]
             |> Set.ofSeq
-            
+
+        let nowarn =
+            let createHashDirective directive values  =
```

Comment:
@7sharp9 	i added this helper, will be completly replaced when PR https://github.com/ctaggart/FsAst/pull/13 is merged and a new `FsAst` is released

Reply:
Cool, fsast just makes it easier to add to the syntax tree via updating default records and settings.

Reply:
yes, seen the why is useful, nice lib.